src/draw/sw/blend/lv_draw_sw_blend_to_rgb888.c

```diff
@@ -922,6 +922,11 @@ static inline void LV_ATTRIBUTE_FAST_MEM blend_non_normal_pixel(uint8_t * dest,
             res[1] = (dest[1] * src.green) >> 8;
             res[2] = (dest[2] * src.red) >> 8;
             break;
+        case LV_BLEND_MODE_DIFFERENCE:
+            res[0] = LV_ABS(dest[0] - src.blue);
```

Comment:
I'm not sure if it's needed, but it'd make sense as the the numbers are unsigned by default, so `1 - 2 = 254`
```suggestion
            res[0] = LV_ABS((int16_t)dest[0] - src.blue);
```

Reply:
Integer promotions happen automatically in C when performing arithmetic operations, but maybe we can add it?

```
uint8_t x = 1;
uint8_t y = 2;
uint8_t res = LV_ABS(x - y);
LV_LOG_USER("Without cast 1 - 2 = %d", res);
res = LV_ABS((int16_t)x - y);
LV_LOG_USER("With cast 1 - 2 = %d", res);
```

> [User]  (0.496, +496)    main: Without cast 1 - 2 = 1 main.cpp:327
> [User]  (0.496, +0)      main: With cast 1 - 2 = 1 main.cpp:329